packages/medusa/src/services/__mocks__/cart.ts

```diff
@@ -349,15 +351,18 @@ export const CartServiceMock = {
         profile_id: "default_profile",
       }
     }
-    if (optionId === IdMap.getId("withData")) {
-      return {
-        id: IdMap.getId("withData"),
-        profile_id: "default_profile",
-        data: {
-          some_data: "yes",
-        },
-      }
+
+    // RW 03/08/2022
```

Comment:
Typescript was complaining that "not all code paths return an object" so I've made the second `if` the default return.